signing-debug.properties

```diff
@@ -0,0 +1,4 @@
+storeFile=../debug.keystore
```

Comment:
if I ever release this app, I'll need to add a new signing key (secret!) and then update the `assetlinks.json` to include the SHA256 of the release key too, else app links won't work.